### ISSUE TITLE
Discovery & Access User Reps Meeting 

### ISSUE THREAD

**Issue by dbranchini** [cul-it/wp-cul-main#67]

**Comment by sew268** [comment]
@dbranchini do you know who the user reps are?  I think I was waitin on that info...if you don't know, I can out to a few people who might know.

**Comment by mhk33** [comment]
For D&A, you could reach out to Lenora Schneller, she is the lead for the user rep group. The rest of the user reps are Caitlin Finlay, Fred Muratori, Jenn Colt, Nina Scholtz, Shayla Harrington, Pam Stansbury, Eisha Neely and Emma Raub.

On a side note, I would love to see the catalog-related feedback from SLAC. I keep meaning to take a look at the notes.

**Comment by dbranchini** [comment]
@mhk33, thank for your providing the contact for @sew268 above. Here's the raw notes from SLAC - https://cornell.box.com/s/3wn0r4v5t6gvsg5co44xsafywj5i3f00. And here's the key takeaways: #46. 

Bonna asked me to share these with Lenora too so I'm going to share these with you, Adam and Lenora via email.

**Comment by dbranchini** [comment]
Also @mhk33 we're hoping to have a discussion with you and the user reps group about possibly considering an integrated approach between the main site and the catalog. One thing that you'll see in our notes is that when users were describing something to us, they frequently didn't know they left the main site and were on the catalog site, and probably vice versa, although many users come to the main site to initiate a catalog search. Of course, it's not important that our users know the URL, but it might be nice if we made those sites more tightly integrated. I'm not sure if we can accomplish that in the same meeting that Sara is planning (probably not). Would you like to come to a main site redesign meeting to discuss this without the user reps first? Or do you prefer to have that conversation with their user reps and a couple of reps from our group?

**Comment by dbranchini** [comment]
@sew268 I'd love to come to this meeting when you plan it.

**Comment by mhk33** [comment]
When we did the last redesign it was in coordination with implementing the Blacklight catalog, so the two sites were done together, and the main site was sort of (for a little while) under the purview of D&A -- at least the home page single search box area. So I think it makes sense to have the groups be on the same page. I don't think I need to come to a redesign meeting first, before you meet with D&A user reps, unless you think that would be helpful in preparation for the meeting!

**Comment by sew268** [comment]
@dbranchini you bet!  It will be great to have you there.  I'll reach out to Lenora now and see when we can get something scheduled...will keep you posted.

**Comment by manolobevia** [comment]
@sew268 I would like to attend too if possible.

**Comment by sew268** [comment]
@manolobevia absolutely!  I've written to Lenora and I'll let everyone know when we've got a date/time set.

**Comment by manolobevia** [comment]
@sew268 THANK YOU!
